release/Makefile

```diff
@@ -0,0 +1,107 @@
+# Needed for glob expansion.
+SHELL = /bin/bash
+.SHELLFLAGS = -O extglob -c
+
+GIT_VERSION = $(shell git describe --dirty --always --tags)
+ifneq ($(DRY_RUN),)
+SNAPSHOT_FLAGS = --snapshot --skip-publish --rm-dist
```

Comment:
Looks like a goreleaser thing snuck into this PR. Is that intended?

Reply:
Nope, removing.